cpu/esp32/periph/timer.c

```diff
@@ -74,6 +73,7 @@
 #define HW_TIMER_CLK_DIV      (rtc_clk_apb_freq_get() / 1000000)
 #define HW_TIMER_CORRECTION   (RTC_PLL_320M / CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ)
 #define HW_TIMER_DELTA_MIN    (MAX(HW_TIMER_CORRECTION << 1, 5))
+#define HW_TIMER_FREQUENCY    (1000000UL) /* only 1Mhz is supported */
```

Comment:
Maybe a bit nitpicking :wink: 
```suggestion
#define HW_TIMER_FREQUENCY    (1000000UL) /* only 1MHz is supported */
```